com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.RenderGraph.cs

```diff
@@ -65,113 +188,235 @@ class AlphaCopyPassData
-            //                    {
-            //                        DoHistogramBasedExposure(cmd, camera, source);
-            //                    }
-            //                    else
-            //                    {
-            //                        DoDynamicExposure(cmd, camera, source);
-            //                    }
-
-            //                    // On reset history we need to apply dynamic exposure immediately to avoid
-            //                    // white or black screen flashes when the current exposure isn't anywhere
-            //                    // near 0
-            //                    if (camera.resetPostProcessingHistory)
-            //                    {
-            //                        var destination = m_Pool.Get(Vector2.one, m_ColorFormat);
-
-            //                        var cs = m_Resources.shaders.applyExposureCS;
-            //                        int kernel = cs.FindKernel("KMain");
-
-            //                        // Note: we call GetPrevious instead of GetCurrent because the textures
-            //                        // are swapped internally as the system expects the texture will be used
-            //                        // on the next frame. So the actual "current" for this frame is in
-            //                        // "previous".
-            //                        cmd.SetComputeTextureParam(cs, kernel, HDShaderIDs._ExposureTexture, GetPreviousExposureTexture(camera));
-            //                        cmd.SetComputeTextureParam(cs, kernel, HDShaderIDs._InputTexture, source);
-            //                        cmd.SetComputeTextureParam(cs, kernel, HDShaderIDs._OutputTexture, destination);
-            //                        cmd.DispatchCompute(cs, kernel, (camera.actualWidth + 7) / 8, (camera.actualHeight + 7) / 8, camera.viewCount);
-
-            //                        PoolSource(ref source, destination);
-            //                    }
-            //                }
-            //            }
-
             if (m_PostProcessEnabled)
             {
-                //                // Temporal anti-aliasing goes first
-                //                bool taaEnabled = false;
+                // Guard bands (also known as "horrible hack") to avoid bleeding previous RTHandle
+                // content into smaller viewports with some effects like Bloom that rely on bilinear
+                // filtering and can't use clamp sampler and the likes
+                // Note: some platforms can't clear a partial render target so we directly draw black triangles
+                using (var builder = renderGraph.AddRenderPass<GuardBandPassData>("Guard Band Clear", out var passData, ProfilingSampler.Get(HDProfileId.GuardBandClear)))
+                {
+                    passData.source = builder.ReadTexture(source);
```

Comment:
Shouldn't this be WriteTexture?

Reply:
Yup, my mistake (my brain saw source and put read :D )